src/api/tests/fixtures/revisions_setup.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from api.models import CollectRecord, Revision
+
+
+@pytest.fixture
+def collect_record_revision_with_updates(db, project1, profile1):
+    collect_record = CollectRecord.objects.create(
+        project=project1, profile=profile1, data=dict()
+    )
+
+    collect_record.save()
+    
+    CollectRecord.objects.create(
+        project=project1, profile=profile1, data=dict()
+    )
+    
+    rev_rec = Revision.objects.get(record_id=collect_record.id)
+    rev_rec.id
```

Comment:
What does this do?

Reply:
populates the rev_rec object before collect record gets deleted (a couple lines down) otherwise the rev_rec values will be different.

I added a note to the code of what's going on